extensions/wikibase/module/styles/schema-alignment.css

```diff
@@ -66,16 +66,31 @@
   margin: 1em;
 }
 
-.schema-alignment-save {
+.schema-alignment-save-a {
```

Comment:
This class name is not very clear - I am not sure what the `-a` suffix refers to.

Reply:
Originally, the buttons 'Save schema' and 'Discard changes' were placed one below the other. But to make them horizontally aligned, I created another class for each: ' schema-alignment-save-a' and 'schema-alignment-save-b', both floating right.
This way, I found 'Save schema' and 'Discard changes' were horizontally aligned towards the right in the hierarchy mentioned in the issue (XD image).
Is there a better and more efficient method to handle this?

Reply:
How about one div containing both buttons, positioned in the right place, and then the buttons inside it one after the other?

Reply:
Is this what you mean?
![image](https://user-images.githubusercontent.com/108210025/224541987-7102441f-5563-4f6e-bd41-601f0079220b.png)

Reply:
Yes! I assume you might want some class name on the containing `div` so that you can position it the right way. Also, `schema-alignment-save` seems like it should not apply to the `discardButton`, but that's a detail.

Reply:
The recent commit seems to be working. 
Outer div is called `schema-alignment-save-buttons` which consists of both the buttons 'save schema' and 'discard changes'.